lib/solidus_usps/calculator/media_mail.rb

```diff
@@ -0,0 +1,31 @@
+module SolidusUsps
+  module Calculator
+    class MediaMail < SolidusUsps::Calculator::Base
+      MEDIA_MAIL_CATEGORY_NAME = "Media Mail".freeze
```

Comment:
Nit: This constant is defined the `MediaMail` class so `MEDIA_MAIL_` is a redundant prefix in the end. Suggestion:

```diff
- SolidusUsps::Calculator::MediaMail::MEDIA_MAIL_CATEGORY_NAME
+ SolidusUsps::Calculator::MediaMail::CATEGORY_NAME
```